src/sentry/eventstream/kafka/backend.py

```diff
@@ -148,6 +148,18 @@ def _send(
             # flush() is a convenience method that calls poll() until len() is zero
             self.producer.flush()
 
+    @staticmethod
+    def _key(headers: Optional[Mapping[str, str]], project_id: str) -> Optional[str]:
+        if "transaction_forwarder" in headers:
```

Comment:
Why shouldn't this happen if transaction_forwarder is not in the headers ?

Reply:
what `message_type` would you assume if `transaction_forwarder` is not in the headers?